libr/include/r_endian.h

```diff
@@ -0,0 +1,211 @@
+#ifndef R_ENDIAN_H
+#define R_ENDIAN_H
+
+/* Endian agnostic functions working on single byte. */
+
+static inline ut8 read_ble8(const void *src) {
```

Comment:
we should prefix all those global functions with `r_` something to avoid namespace collissions. proposals for this?

`r_read_ble8` ? `r_endian_read_ble8` looks too long to me